src/main/java/jenkins/plugins/nodejs/configfiles/Npmrc.java

```diff
@@ -141,6 +142,8 @@ public String toString() {
      * @throws IOException in case of I/O write error
      */
     public void save(File file) throws IOException {
+        // FileWriterWithEncoding does not truncate file if it already exists
+        new FileOutputStream(file).close();
```

Comment:
instead to rely on the implementation of internal method I would prefer something like
`FileUtils.writeStringToFile(file, toString(), UTF_8);` to archive the same goal but in a more readable way.